### PR TITLE
[percona] up to 5.6.43

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: library/percona [hunk 1]
```diff
@@ -14,8 +14,8 @@ GitCommit: ecfd4173d627d5b5548e8059d1828a09af867059
 Directory: percona-server.57
 File: Dockerfile-dockerhub
 
-Tags: 5.6.42-centos, 5.6-centos, 5.6.42, 5.6, ps-5.6.42, ps-5.6
-GitCommit: 290549e8e7ac5acbe4a3aa76ce282bc5e0e21043
+Tags: 5.6.43-centos, 5.6-centos, 5.6.43, 5.6, ps-5.6.43, ps-5.6
+GitCommit: 10b956ffde7ac157a9e87f779861631e6617296c
 Directory: percona-server.56
 File: Dockerfile-dockerhub
```
Comment (tianon) [inline library/percona]: This file at this commit still has `ENV PERCONA_VERSION 5.6.42-rel84.2.el7`, which is definitely not going to give us 5.6.43. :sweat_smile:

Comment (EvgeniyPatlan) [reply library/percona]: @tianon thank you! I have fixed commit id!

Comment (tianon) [reply library/percona]: Are you sure? https://github.com/percona/percona-docker/blob/dee3f651880daa72a610bda18c4e2fc3fe3250a4/percona-server.56/Dockerfile-dockerhub#L19 :smile:

(that line is the same on the specified commit and on master)

Comment (EvgeniyPatlan) [reply library/percona]: @tianon I have understand what was the issue. Now it is fixed. Thanks for your help!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 1bdd41a..d645ee9 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -2,8 +2,8 @@ percona:5
 percona:5-centos
 percona:5.6
 percona:5.6-centos
-percona:5.6.42
-percona:5.6.42-centos
+percona:5.6.43
+percona:5.6.43-centos
 percona:5.7
 percona:5.7-centos
 percona:5.7.24
@@ -20,7 +20,7 @@ percona:psmdb-3.6
 percona:psmdb-3.6.8
 percona:ps-5
 percona:ps-5.6
-percona:ps-5.6.42
+percona:ps-5.6.43
 percona:ps-5.7
 percona:ps-5.7.24
 percona:ps-8
diff --git a/percona_ps-5.6/Dockerfile-dockerhub b/percona_ps-5.6/Dockerfile-dockerhub
index 2f2d41d..df02025 100644
--- a/percona_ps-5.6/Dockerfile-dockerhub
+++ b/percona_ps-5.6/Dockerfile-dockerhub
@@ -10,13 +10,16 @@ RUN export GNUPGHOME="$(mktemp -d)" \
 	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
 	&& gpg --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona \
 	&& rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 \
-	&& curl -L -o /tmp/percona-release.rpm http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-6.noarch.rpm \
+	&& curl -L -o /tmp/percona-release.rpm https://repo.percona.com/centos/7/RPMS/noarch/percona-release-0.1-8.noarch.rpm \
 	&& rpmkeys --checksig /tmp/percona-release.rpm \
 	&& yum install -y /tmp/percona-release.rpm \
-	&& rm -rf "$GNUPGHOME" /tmp/percona-release.rpm
+	&& rm -rf "$GNUPGHOME" /tmp/percona-release.rpm \
+	&& rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY \
+	&& percona-release disable all \
+	&& percona-release enable percona release
 
 # install exact version of PS for repeatability
-ENV PERCONA_VERSION 5.6.42-rel84.2.el7
+ENV PERCONA_VERSION 5.6.43-rel84.3.el7
 
 RUN yum install -y \
 		Percona-Server-server-56-${PERCONA_VERSION} \
```

</details>

**Comment by yosifkit** [comment]
Build test of #5359; 4685c7ab; `amd64` (`percona`):

```console

$ bashbrew build percona:8.0.13-4-centos
Using bashbrew/cache:98768e369406bcff501efd808414f1c2a3f56732ca10b0fde7b31d384487e0ea (percona:8.0.13-4-centos)
Tagging percona:8.0.13-4-centos
Tagging percona:8.0-centos
Tagging percona:8-centos
Tagging percona:8.0.13-4
Tagging percona:8.0
Tagging percona:8
Tagging percona:ps-8.0.13-4
Tagging percona:ps-8.0
Tagging percona:ps-8

$ test/run.sh percona:8.0.13-4-centos
testing percona:8.0.13-4-centos
	'utc' [1/9]...passed
	'cve-2014--shellshock' [2/9]...passed
	'no-hard-coded-passwords' [3/9]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/9]...passed
	'mysql-basics' [5/9].......passed
	'mysql-initdb' [6/9].......passed
	'mysql-log-bin' [7/9].......passed
	'percona-tokudb' [8/9].......passed
	'percona-rocksdb' [9/9].......passed


$ bashbrew build percona:5.7.24-centos
Using bashbrew/cache:3395ce51f8f9e8ceeccb7c5efbbf79f9822f962d7dedeb7a8c558e6fa27cd991 (percona:5.7.24-centos)
Tagging percona:5.7.24-centos
Tagging percona:5.7-centos
Tagging percona:5-centos
Tagging percona:centos
Tagging percona:5.7.24
Tagging percona:5.7
Tagging percona:5
Tagging percona:ps-5.7.24
Tagging percona:ps-5.7
Tagging percona:ps-5
Tagging percona:latest

$ test/run.sh percona:5.7.24-centos
testing percona:5.7.24-centos
	'utc' [1/9]...passed
	'cve-2014--shellshock' [2/9]...passed
	'no-hard-coded-passwords' [3/9]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/9]...passed
	'mysql-basics' [5/9]......passed
	'mysql-initdb' [6/9].......passed
	'mysql-log-bin' [7/9].......passed
	'percona-tokudb' [8/9].......passed
	'percona-rocksdb' [9/9]......passed


$ bashbrew build percona:5.6.43-centos
Building bashbrew/cache:6f08f29705d02102b590834059394f144cb79c485eff10c59c0ec9e4bcec485d (percona:5.6.43-centos)
Tagging percona:5.6.43-centos
Tagging percona:5.6-centos
Tagging percona:5.6.43
Tagging percona:5.6
Tagging percona:ps-5.6.43
Tagging percona:ps-5.6

$ test/run.sh percona:5.6.43-centos
testing percona:5.6.43-centos
	'utc' [1/9]...passed
	'cve-2014--shellshock' [2/9]...passed
	'no-hard-coded-passwords' [3/9]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/9]...passed
	'mysql-basics' [5/9].......passed
	'mysql-initdb' [6/9]........passed
	'mysql-log-bin' [7/9].......passed
	'percona-tokudb' [8/9]........passed
	'percona-rocksdb' [9/9]...passed


$ bashbrew build percona:psmdb-3.6.8
Using bashbrew/cache:0c6b2b6c8e28a286afb42fb2c0ef00500cd681df7c11e7beaa9e8a4fbbc1c702 (percona:psmdb-3.6.8)
Tagging percona:psmdb-3.6.8
Tagging percona:psmdb-3.6

$ test/run.sh percona:psmdb-3.6.8
testing percona:psmdb-3.6.8
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/6]...passed
	'mongo-basics' [5/6]....passed
	'mongo-auth-basics' [6/6]....passed

```

**Pull request merged**